test/test_output_engine.py

```diff
@@ -49,8 +49,8 @@ class TestOutputEngine(unittest.TestCase):
             "comments": "",
         },
         {
-            "vendor": "vendorname0",
-            "product": "productname0",
+            "vendor": "vendor0",
+            "product": "product0",
             "version": "1.0",
             "cve_number": "CVE-1234-9876",
```

Comment:
```suggestion
            "cve_number": "CVE-1234",
```
Also, add a shorter CVE Number

Reply:
Yes, sure. There was another subtle thing I missed out. When I shortened `vendor` and `product` names, I missed out on changing the length of the dashes at the end of `expected output`. I corrected it. I think this time the tests will pass.